Makefile

```diff
@@ -20,16 +20,26 @@ help:
 	@echo "  test_frontend   : Run  frontend tests using selenium (make sure node is installed first)"
 	@echo ""
 
-up:
-	docker compose up -d
+unseal:
+	echo please use key to run the project
+	@docker exec -it transcendence-vault-1 bash
```

Comment:
Are you sure about using bash for an infinite loop?

Reply:
Bash here is not used for infinite loop, it's used to prompt for unseal keys